packages/contracts-bedrock/package.json

```diff
@@ -15,6 +15,7 @@
   ],
   "scripts": {
     "build:forge": "forge build",
+    "build:with-metadata": "hardhat clean && FOUNDRY_PROFILE=echidna hardhat compile",
```

Comment:
Echidna needs the metadata hashes to work properly. The `FOUNDRY_PROFILE` flag is natively recognized by the `foundry.toml` file, and it's been manually added to have the same effect in the hardhat config file.